tests/containers/run_container_in_k3s.pm

```diff
@@ -16,8 +16,20 @@ use serial_terminal 'select_serial_terminal';
 use utils;
 use version_utils;
 use publiccloud::utils;
+use containers::utils qw(check_min_runtime_version);
+use Utils::Architectures qw(is_ppc64le);
 use containers::k8s qw(install_k3s uninstall_k3s apply_manifest wait_for_k8s_job_complete find_pods validate_pod_log);
 
+sub prep_pod_yaml {
```

Comment:
Why not prepare_pod_yaml ?

Reply:
Implemented the suggestion to make it more comprehensible.

Reply:
I'm also not a big fan of abbrevating-single-words-because-we're-too-lazy-to-type.